Rethinking_2/Chp_08.ipynb

```diff
@@ -0,0 +1,1578 @@
+{
```

Comment:
<p>use <code>var_names</code>  instead of <code>vars</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>